### DISCUSSION TITLE
New commit and activity colours

### DISCUSSION THREAD

**Discussion by matievisthekat** [community/community#7062] [Other Feature Feedback, Questions, & Ideas] [answered]
I'm not sure if this is a global change, but for me, the activity colours have changed from a green:

![Screenshot from 2021-10-31 09-30-46](https://user-images.githubusercontent.com/45036977/139573112-ba0529bb-bc71-4de5-a38b-75708d17f5c0.png)

to an orange/yellow

![Screenshot from 2021-10-31 09-30-08](https://user-images.githubusercontent.com/45036977/139573115-bb116b2f-6bf5-4d48-9d4e-96fb7a2a6fa4.png)

Surprisingly the orange doesn't look too bad, but I think the green was a better colour

**Comment by Katsute** [comment, marked as answer]
It's orange because today is Halloween, it should be back to green tomorrow.

**Comment by norwd** [comment]
This there a way to disable the change? Or to opt out of GH making visual changes like this? I just spent 20-odd minutes trying to debug the colour blind themes to figure out why my contributions were suddenly invisible 😕

Edit: clarification, I'm not colour-blind, I just use those themes for aesthetic reasons.

**Reply by scarletquasar** [reply]
Any update on that? No way they let a hardcoded option against people will and no path to disable it
